gn3/function_helpers.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
**Nitpick:** I generally avoid, if I can, *utility* or *helper* modules. These usually grow to be a space where someone, less aversed with the codebase, dump functions/ code they can't place elsewhere. If we have them, I try to keep them as slim as possible.